src/python_testing/TC_RVCOPSTATE_2_5.py

```diff
@@ -0,0 +1,273 @@
+#
+#    Copyright (c) 2025 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# === BEGIN CI TEST ARGUMENTS ===
+# test-runner-runs:
+#   run1:
+#     app: ${CHIP_RVC_APP}
+#     app-args: --discriminator 1234 --KVS kvs1 --trace-to json:${TRACE_APP}.json --app-pipe /tmp/rvcopstate_2_5_fifo
+#     script-args: >
+#       --storage-path admin_storage.json
+#       --commissioning-method on-network
+#       --discriminator 1234
+#       --passcode 20202021
+#       --trace-to json:${TRACE_TEST_JSON}.json
+#       --trace-to perfetto:${TRACE_TEST_PERFETTO}.perfetto
+#       --endpoint 1
+#       --app-pipe /tmp/rvcopstate_2_5_fifo
+#       --PICS examples/rvc-app/rvc-common/pics/rvc-app-pics-values
+#       --int-arg runmode_cleanmode:1
+#     factory-reset: true
+#     quiet: true
+# === END CI TEST ARGUMENTS ===
+
+import asyncio
+import enum
+import logging
+
+import chip.clusters as Clusters
+from chip.testing.event_attribute_reporting import ClusterAttributeChangeAccumulator
+from chip.testing.matter_testing import (AttributeMatcher, MatterBaseTest, TestStep, async_test_body, default_matter_test_main,
+                                         type_matches)
+from mobly import asserts
+
+
+class RvcStatusEnum(enum.IntEnum):
+    # TODO remove this class once InvalidInMode response code is implemented in python SDK
+    Success = 0x0
+    UnsupportedMode = 0x1
+    GenericFailure = 0x2
+    InvalidInMode = 0x3
+
+
+def error_enum_to_text(error_enum):
```

Comment:
does an enum class not have a readable `%` to be used instead? Having this seems harder to maintain.

Or maybe override the __str__ method if we really need to...

<img width="384" height="315" alt="image" src="https://github.com/user-attachments/assets/126fcb40-5da3-471b-83da-c7d46fa5cb3b" />